kamelets/aws-ddb-sink.kamelet.yaml

```diff
@@ -97,6 +97,12 @@ spec:
         x-descriptors:
           - 'urn:alm:descriptor:com.tectonic.ui:checkbox'
         default: false
+      inputFormat:
```

Comment:
the list of possible types must be expressed through an enum

Reply:
same as for output formats. I think the user may provide a custom format, too.

Reply:
yes, but at least as description or as information, we need a list of default types.

Reply:
@oscerd valid concern and I think we can add a list of default types. this is to be done in a separate PR though once Camel K Kamelet CRDs are updated with additional data type fields.